framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/InfoPanel/InfoPanel.module.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../styles/constants';
+
+.infoHeading {
```

Comment:
This is also not required if the Text component is used

Reply:
same here. The Text component should be extended to be used here

Reply:
im not sure what do u mean by extending here, but 
```
{props.title}
```
this can be just
```
<Text type="tr" color={props.color}>{props.title}</Text>
```
(type tr probably better to be renamed in other issue)

Reply:
This doesn't work because in the "InfoPanel" with `icon` "margins" are changed and it is not styled similar to other panels.

Reply:
you can keep the margins for outer layer, but the Text related styles can be just removed from here as the responsibility is in `Text` Component